js/web/lib/wasm/jsep/webgpu/ops/3rd-party/matmul_packed_webgpu.ts

```diff
@@ -40,6 +39,7 @@ import {
   getActivationSnippet,
   InternalActivationAttributes,
 } from '../fuse-utils';
+import { convertOutputBatchIndicesToInputBatchIndices } from '../matmul';
```

Comment:
Will it have potential issue if we import something from `matmul.ts`. Meanwhile, we also import something else from `matmul_packed_webgpu.ts`? It will have circle from files, but import different functions. Just curious whether it brings any problem. @fs-eire.

Others LGTM, thanks.

Reply:
In general we should avoid circular dependency. we can create a `matmul-shared.ts` and move some of the dependencies there to avoid this.

Reply:
Done, separate shader and helper implementation from matmul.ts now. Please take a look, thanks!